docker-compose.minimal.yml

```diff
@@ -521,7 +526,6 @@ services:
     depends_on:
       - jaeger
     logging: *logging
-
```

Comment:
Could you also add the line break back here?

Reply:
sure